static/src/js/components/Preferences/__tests__/PreferencesForm.test.jsx

```diff
@@ -1,70 +1,79 @@
 import mapLayers from '../../../constants/mapLayers'
 import projectionCodes from '../../../constants/projectionCodes'
+import setupTest from '../../../../../../jestConfigs/setupTest'
+import useEdscStore from '../../../zustand/useEdscStore'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-const defaultProps = {
+const defaultZustandState = {
   preferences: {
     preferences: {
       panelState: 'default',
       collectionListView: 'default',
-      granuleListView: 'default'
+      granuleListView: 'default',
+      collectionSort: 'default',
+      granuleSort: 'default',
+      mapView: {
+        baseLayer: 'worldImagery',
+        latitude: 0,
+        longitude: 0,
+        overlayLayers: [
+          'bordersRoads',
+          'placeLabels'
+        ],
+        projection: 'epsg4326',
+        rotation: 0,
+        zoom: 3
+      }
     },
-    isSubmitting: false
-  },
-  onUpdatePreferences: jest.fn()
-}
-
-function setup() {
-  const enzymeWrapper = shallow(<PreferencesForm {...defaultProps} />)
-
-  return {
-    enzymeWrapper,
-    props: defaultProps
+    isSubmitting: false,
+    submitAndUpdatePreferences: jest.fn()
   }
 }
 
-function setupMount() {
-  const enzymeWrapper = mount(<PreferencesForm {...defaultProps} />)
-
-  return {
-    enzymeWrapper,
-    props: defaultProps
-  }
-}
+const setup = setupTest({
+  Component: PreferencesForm,
+  defaultZustandState
+})
 
 describe('PreferencesForm component', () => {
-  test('renders a Form component', () => {
-    const { enzymeWrapper, props } = setup()
+  beforeEach(() => {
+    useEdscStore.setState(useEdscStore.getInitialState(), true)
+  })
 
-    const form = enzymeWrapper.find(Form)
+  test('renders a Form component', () => {
+    setup()
 
-    expect(form.props().schema).toEqual(schema)
-    expect(form.props().uiSchema).toEqual(uiSchema)
-    expect(form.props().formData).toEqual(props.preferences.preferences)
-    expect(form.props().onSubmit).toEqual(props.onUpdatePreferences)
+    expect(screen.getByRole('button', { name: /submit/i })).toBeInTheDocument()
+    expect(screen.getByText(/Panel State/i)).toBeInTheDocument()
+    expect(screen.getByText(/Collection.*Sort/i)).toBeInTheDocument()
+    expect(screen.getByText(/Granule.*Sort/i)).toBeInTheDocument()
   })
 
-  test('onChange sets the state', () => {
-    const { enzymeWrapper } = setupMount()
+  test('onChange sets the state', async () => {
+    setup()
+
+    const submitButton = screen.getByRole('button', { name: /submit/i })
 
-    act(() => {
-      enzymeWrapper.find(Form).props().onChange({
-        formData: {
+    expect(submitButton).toBeInTheDocument()
+    expect(screen.getByText(/Panel State/i)).toBeInTheDocument()
+    expect(screen.getByText(/Collection.*Sort/i)).toBeInTheDocument()
+    expect(screen.getByText(/Granule.*Sort/i)).toBeInTheDocument()
+
+    expect(submitButton).not.toBeDisabled()
+  })
+
+  test('updating store data updates the state', () => {
```

Comment:
This test isn't testing that anything has been updated, it has the same issues as the first test. 

But I don't think this test is necessary. If the second test performs an action and verifies the form data is updated, then that covers all of the logic in the component.